app/serializers/basic_post_serializer.rb

```diff
@@ -46,7 +46,6 @@ def cooked
   def ignored
     object.is_first_post? &&
       scope.current_user&.id != object.user_id &&
-      !object.user&.staff? &&
```

Comment:
Come to think about it, this is not needed here.

Reply:
Right because it was enforced when the `IgnoredUser` was created?

Reply:
I think we should keep it since it'll avoid making the SQL query when the owner of the 1st is a staff member.

Reply:
This actually is not correct, it is a bug. We are checking here if this Post should be ignored or not. It shouldn't matter if this is a `staff`'s post or not.